src/vue/forms/InvestForm.vue

```diff
@@ -418,9 +488,8 @@ export default {
 
     async submit () {
       if (!this.isFormValid()) return
-
+      this.updateView(VIEW_MODES.submit)
       this.disableForm()
-      this.isSubmitting = true
```

Comment:
why removed? that was nice the the buttons were disabled by the submitting. confirmation buttons should also be disabled

Reply:
returned